### PR TITLE
Add option to change the default height of the rows

### PR DESCRIPTION
**Pull request by gabrielbs**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,26 +1,36 @@
...
+    labelImage: "Image",
+    labelVideo: "Video",
+    labelMap: "Map",
+    rowHeight: 75,
     ...opts
   };
 
   // Add blocks
-  const loadBlocks = require('./blocks');
+  const loadBlocks = require("./blocks");
```
Comment (artf) [inline src/index.js]: All single quotes were changed in double-quotes, is it intended o something related to your editor? In any case, I'd like to keep single quotes

File: src/blocks.js [hunk 2]
```diff
@@ -19,28 +21,31 @@ export default function (editor, opt = {}) {
...
-    }` : `
+    }`
+    : `
     .${clsRow} {
       display: table;
       padding: 10px;
       width: 100%;
     }
-    @media (max-width: 768px) {
+    @media (max-width: 768pxpx) {
```
Comment (artf) [inline src/blocks.js]: should be 768px

**Review by artf — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (gabrielbs) [reply src/index.js]: I'm sorry, I don't realize that my editor was changed for double-quotes.... I just put the single-quotes back

**Comment by gabrielbs** [comment]
hey @artf I just did the changes, could you review again please?

**Review by artf — APPROVED** [review]

**Pull request merged**